<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPasswordDialog</class>
 <widget class="QDialog" name="MainPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Enter password</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="passwordBox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #282d31; 
border-radius: 8px;
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="m_passwordEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>271</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 3px; border: 1px solid #555; border-radius: 5px;   color: #aaa; font-size: 14px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Type your password...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b1_continue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>320</y>
        <width>91</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* This button style is handled automatically */</string>
      </property>
      <property name="text">
       <string>CONTINUE</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_errorLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>122</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: orange; font-size: 14px; border: none;</string>
      </property>
      <property name="text">
       <string>Incorrect Password. Please try again.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dTitle_welcome">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ddd;
border-right: 0px;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>Welcome Back</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_currentWalletTitle">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ccc;
        border-right: 0px;
         border: 0px;
font-size: 14px;</string>
      </property>
      <property name="text">
       <string notr="true">currentWallet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_instructions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>122</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ccc; font-size: 14px; border: none;</string>
      </property>
      <property name="text">
       <string>Click CONTINUE or press ENTER to unlock your wallet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_version">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ccc;
border-right: 0px;
font-size: 14px;</string>
      </property>
      <property name="text">
       <string>Conceal Desktop 6.7.3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="m_helpButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 3px; background-color: #444; border-radius: 5px;   color: #aaa; font-size: 10px;
}
QPushButton:hover {
color: orange
}</string>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="m_quitButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>430</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 3px; background-color: #444; border-radius: 5px;   color: #aaa; font-size: 10px;
}
QPushButton:hover {
color: orange
}</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
padding: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resources.qrc">:/images/conceal-logo</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="m_changeButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>430</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 3px; background-color: #444; border-radius: 5px;   color: #aaa; font-size: 10px;
}
QPushButton:hover {
color: orange
}</string>
      </property>
      <property name="text">
       <string>CHANGE</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_instructions_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>122</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ccc; font-size: 14px; border: none;</string>
      </property>
      <property name="text">
       <string>Click CHANGE to choose (or create) another wallet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b1_continue</sender>
   <signal>clicked()</signal>
   <receiver>MainPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_helpButton</sender>
   <signal>clicked()</signal>
   <receiver>MainPasswordDialog</receiver>
   <slot>helpClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainPasswordDialog</receiver>
   <slot>quitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_changeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainPasswordDialog</receiver>
   <slot>changeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quitClicked()</slot>
  <slot>helpClicked()</slot>
  <slot>changeClicked()</slot>
 </slots>
</ui>
